src/mmw/js/src/modeling/models.js

```diff
@@ -898,7 +898,7 @@ var ScenarioModel = Backbone.Model.extend({
                 // Merge the values that came back from Mapshed with the values
                 // in the modifications from the user.
                 var modifications = self.get('modifications'),
-                    mergedGisData = project.get('gis_data');
+                    mergedGisData = _.cloneDeep(project.get('gis_data'));
```

Comment:
Another way to view the issue, pre-fix, is that after `_.assign`ing to `mergedGisData` below, the contents of `project.get('gis_data')` would also have the updated values.  Since that project data is the base and we just apply modifications on the fly before `gwlfe`, we were corrupting everything.